aspnetcore/mvc/controllers/application-model.md

```diff
@@ -12,7 +12,187 @@ ms.technology: aspnet
 ms.prod: aspnet-core
 uid: mvc/controllers/application-model
 ---
-# 🔧 Working with the Application Model
+# Working with the Application Model
+
+By [Steve Smith](http://ardalis.com)
+
+ASP.NET Core MVC defines an *application model* representing the components of an MVC app. You can read and manipulate this model to modify how MVC elements behave.
+
+## Models and Providers
+
+The ASP.NET Core MVC application models include both abstract interfaces and concrete implementations that define how MVC behaves. This behavior includes how MVC determines and uses controller names, action names, action parameters, routes, and filters. By working with the application model, you can modify your app to follow different conventions from the default MVC behavior.
```

Comment:
> include both abstract interfaces and concrete implementations that define how MVC behaves. This behavior includes how MVC determines and uses controller names, action names, action parameters, routes, and filters

This isn't really accurate. The application model is **data** that contains the results of us discovering your controllers and actions according to the default conventions.

I might also be good to mention that parameters, names, routes, and filters are all **configuration data** for actions and controllers.